doc-content/enterprise-only/admin-and-config/logging-events-database-proc.adoc

```diff
@@ -0,0 +1,78 @@
+[id='logging-events-database-proc']
+= Logging events to database
+
+You can log an event to a database that occurs on runtime in a process instance, node instance or variable instance.
+
+.Procedure
+. Map log classes to the data source that enables the data source to accept the log entries.
+* On {EAP}, edit the data source properties in the `persistence.xml` file.
+[source,code]
```

Comment:
The persistence.xml of KIE Server already contains this so I guess it should be present only in an engine section.

Reply:
Moving to process engine doc